### PR TITLE
ORC-995: Failed to build Java library. Declare dependence orc-shims

### PR DESCRIPTION
**Pull request by guiyanakuang**
### What changes were proposed in this pull request?

Declare dependence orc-shims.
```
[INFO] --- maven-dependency-plugin:3.1.2:analyze-only (default) @ orc-examples ---
[WARNING] Used undeclared dependencies found:
[WARNING]    org.apache.orc:orc-shims:jar:1.8.0-SNAPSHOT:compile
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Apache ORC 1.8.0-SNAPSHOT:
[INFO] 
[INFO] Apache ORC ......................................... SUCCESS [  1.592 s]
[INFO] ORC Shims .......................................... SUCCESS [  1.590 s]
[INFO] ORC Core ........................................... SUCCESS [01:49 min]
[INFO] ORC MapReduce ...................................... SUCCESS [ 35.893 s]
[INFO] ORC Tools .......................................... SUCCESS [ 12.179 s]
[INFO] ORC Examples ....................................... FAILURE [  2.986 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:44 min
[INFO] Finished at: 2021-09-15T10:43:44+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.1.2:analyze-only (default) on project orc-examples: Dependency problems found -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :orc-examples
```

### Why are the changes needed?

Fix Build.

### How was this patch tested?

Pass the CIs.

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
Thank you. Is this `main` only issue?

**Comment by guiyanakuang** [comment]
> Thank you. Is this `main` only issue?

It should be a problem caused by the merge of ORC-910, so I think only main is affected at the moment.

**Comment by guiyanakuang** [comment]
> BTW, I sent you an email ([guiyanakuang@gmail.com](mailto:guiyanakuang@gmail.com)) via Gmail. Did you check it?

Sorry I just saw the email.

File: java/examples/pom.xml [hunk 1]
```diff
@@ -36,6 +36,10 @@
       <groupId>org.apache.orc</groupId>
       <artifactId>orc-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.orc</groupId>
+      <artifactId>orc-shims</artifactId>
+    </dependency>
```
Comment (dongjoon-hyun) [inline java/examples/pom.xml:42]: CI seems to fail by this addition.

**Comment by dongjoon-hyun** [comment]
No problem. I just want to reach out to you. :)

**Comment by dongjoon-hyun** [comment]
BTW, please participate the votes. We have three votes in the dev mailing list. :)

File: java/examples/pom.xml (hunk 1 above)
Comment (guiyanakuang) [reply java/examples/pom.xml:42]: I don't know why, but the current build prompt is the opposite of my environment, and I need some time to make sure to analyze it.

**Comment by guiyanakuang** [comment]
Sorry, orc-example does not directly depend on orc-shims. It's just a problem with my env, I will close this pr.

**Comment by dongjoon-hyun** [comment]
Thank you for the confirming, @guiyanakuang . :)

**Pull request closed without merging**